### PR TITLE
CR-1055356: Provided option to reset temp and power threshold & target values using xbmgmt tool

### PR DESCRIPTION
**Pull request by rajkumar-xilinx**
Added new sysfs node "scaling_reset" under xmc to reset the power & temperature's threshold & target values to default settings.

One can reset clock scaling settings to default values using xbmgmt tool also with "--cs_reset 1"

CR-1055356

Signed-off-by: Rajkumar Rampelli <rajkumar@xilinx.com>

### CODE REVIEW

**Comment by rajkumar-xilinx** [comment]
retest this please

**Comment by abalasa** [comment]
retest this please

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 1]
```diff
@@ -1450,6 +1450,48 @@ static bool scaling_condition_check(struct xocl_xmc *xmc, struct device *dev)
 	return true;
 }
 
+static ssize_t scaling_reset_store(struct device *dev,
+	struct device_attribute *da, const char *buf, size_t count)
+{
+	struct xocl_xmc *xmc = platform_get_drvdata(to_platform_device(dev));
+	u32 val = 0, val2, threshold;
+	bool cs_en;
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: Can you please use meaningful names instead of val / val2.

**Comment by maxzhen** [comment]
> 
> 
> Looks good. We should not ask users to use sysfs node to reset the threshold/target values. We should give an option in xbmgmt to reset these targets.
> @maxzhen , Any comments.

You can add new options to xbmgmt config, I think.

**Comment by chvamshi-xilinx** [comment]
> 
> 
> > Looks good. We should not ask users to use sysfs node to reset the threshold/target values. We should give an option in xbmgmt to reset these targets.
> > @maxzhen , Any comments.
> 
> You can add new options to xbmgmt config, I think.

Thanks @maxzhen . @rajkumar-xilinx , Can you please add an option in xbmgmt config for this

**Comment by maxzhen** [comment]
> 
> 
> > > Looks good. We should not ask users to use sysfs node to reset the threshold/target values. We should give an option in xbmgmt to reset these targets.
> > > @maxzhen , Any comments.
> > 
> > 
> > You can add new options to xbmgmt config, I think.
> 
> Thanks @maxzhen . @rajkumar-xilinx , Can you please add an option in xbmgmt config for this

This is a user visible option (v.s. an option for debug purpose), right? In that case, you'd want to work with @rozumx and @AShivangi to add it to xbmgmt2 and have it approved by WTS/PRT.

**Comment by maxzhen** [comment]
@rajkumar-xilinx , please fix your pull request to use standard title and commit msg.

**Comment by rajkumar-xilinx** [comment]
> @rajkumar-xilinx , please fix your pull request to use standard title and commit msg.

Hi @maxzhen , I updated commit message, can you please review once again and let me know if it is right. If not, please give me an example of standard title/commit message. Thank you.

**Pull request merged**